File: web3rpc/rpc-specs/code-samples/python/debug/logging/verbosityByID.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

verId = 1
level = 3

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.verbosity_by_id(verId, level)

print(debug_response)

File: web3rpc/rpc-specs/code-samples/javascript/debug/logging/verbosityByID.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const id = 1;
  const level = 3;

  provider.debug
    .verbosityByID(id, level, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
